### DISCUSSION TITLE
Flash emulator

### DISCUSSION THREAD

**Discussion by delavanty** [Swordfish90/Lemuroid#832] [Ideas]
I was thinking .... Is it possible to make an emulator feature to run old flash games? I know that would be a huge process since flash is considered private software and there is only one software currently available on Google play store that runs on Android 11 decently.... But it has no button controls hence why I was asking big it be possible to add flash to the emulator app as a support. I loved them old flash games like the bubble tanks and bubble Tower

**Comment by Algiuxs** [comment]
It is possible, with ruffle, but I don't think people will implement it.
